CMakeLists.txt

```diff
@@ -112,6 +112,20 @@ endif()
 add_definitions(-Wall -Wextra -pedantic -Wno-unused-parameter
     -Wstrict-prototypes -std=gnu99)
 
+include(CheckCCompilerFlag)
+check_c_compiler_flag(-fstack-protector-strong HAS_FSTACK_PROTECTOR_STRONG_FLAG)
+check_c_compiler_flag(-fstack-protector HAS_FSTACK_PROTECTOR_FLAG)
+
+if(HAS_FSTACK_PROTECTOR_STRONG_FLAG)
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fstack-protector-strong")
+elseif(HAS_FSTACK_PROTECTOR_FLAG)
+  message(STATUS " -fstack-protector-strong is unsupported, falling back to -fstack-protector")
```

Comment:
Sorry, I meant that the extra messages should be dropped (as Justin suggested).  I mentioned status so that you knew about it for future stuff. :-)  Sorry for the confusion.